django/utils/html.py

```diff
@@ -77,6 +78,28 @@ def escapejs(value):
     return mark_safe(str(value).translate(_js_escapes))
 
 
+_json_tag_escapes = {
```

Comment:
I'd still like to see an argument here why we replace those, the reference to http://api.rubyonrails.org/classes/ERB/Util.html#method-c-json_escape does not really count because they insert it in a normal script tag which probably has different parsing rules again. Following the argument in https://github.com/django/django/pull/9235#issuecomment-337607943 I'd also escape single quotes then. Do we have any easily readable document on how browsers are supposed to parse that :/